pkg/operator/sync.go

```diff
@@ -622,6 +625,11 @@ func (optr *Operator) syncCustomResourceDefinitions() error {
 	}
 
 	for _, crd := range crds {
+		DNE := false
+		current, err := optr.ccLister.Get("machine-config-controller")
+		if err != nil && apierrors.IsNotFound(err) {
```

Comment:
suggestion: You can just do `if apierrors.IsNotFound(err)` here. That function performs a nil check and will return false if err is nil.